### PR TITLE
Re-enable ui tests

### PR DESCRIPTION
**Pull request by acosta434**
Re-enable tests to see if they run in current pipeline

Description: 

- Removed ignore from existing tests
- Update string to fix failing test
- Add step to dismiss ArCoredialog

Screenshots: 
n/a

- [ ] Acceptance Criteria satisfied
- [ ] Regression Testing

### CODE REVIEW

File: app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt [hunk 1]
```diff
@@ -77,6 +79,8 @@ open class BaseTest {
         // Catch the exception if the popup doesn't appear so we don't fail the tests
         try {
             device.wait(Until.findObject(By.text("CONTINUE")), 10000).click()
+            Thread.sleep(2000)
+            device.pressBack()
```
Comment (PaulKlauser) [inline app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt:83]: Do we need to press continue, and then back? I think we can just press back and the dialog will dismiss.

File: app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt [hunk 2]
```diff
@@ -46,10 +48,10 @@ open class BaseTest {
         // Since the build machine gets wiped after every run we can check the file storage
         // of the emulator to determine if this is a first time launch
         // and dismiss the full screen immersive popup on android if it is
-        firstLaunch = getInstrumentation().targetContext.filesDir.listFiles().isEmpty()
+/*        firstLaunch = getInstrumentation().targetContext.filesDir.listFiles().isEmpty()
         if (firstLaunch) {
             takeScreenshot("InitialSetup")
-        }
+        }*/
```
Comment (PaulKlauser) [inline app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt:54]: Is this intentionally commented out? If so, we should delete the code. We can bring it back through git if we need to.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -6,6 +6,9 @@
     <uses-permission android:name="android.permission.CAMERA" />
 
     <uses-feature android:name="android.hardware.camera.ar" />
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```
Comment (PaulKlauser) [inline app/src/main/AndroidManifest.xml:11]: Was this change required to get the tests running?

**Review by PaulKlauser — CHANGES_REQUESTED** [review]
Looking good, see my comments. Can we squash these commits down as well?

File: app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt (hunk 1 above)
Comment (acosta434) [reply app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt:83]: No we do not. Pressing click just adds complexity. Removing the 'click' action also gets rid of the need to wait in the next step. Thanks

File: app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt (hunk 2 above)
Comment (acosta434) [reply app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt:54]: That was part of a different problem. Removed.

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (acosta434) [reply app/src/main/AndroidManifest.xml:11]: It is not required right now, removing it.

**Comment by acosta434** [comment]
@PaulKlauser I got it down to the 1 commit.

**Review by PaulKlauser — APPROVED** [review]
